app/controllers/tpi/sectors_controller.rb

```diff
@@ -71,7 +71,7 @@ def user_download_all
     end
 
     def user_download
-      companies_ids = @sector.companies.published.select(:id)
+      companies_ids = @sector.companies.published.active.select(:id)
```

Comment:
Just curious - are we sure that scopes `published` and `active` are independent of each other? Or maybe inactive companies should be always unpublished?

If those are unrelated we could at least add another scope `published_active` as it looks those 2 go together at least in this controller.

Reply:
Active flag is different from the publishable status. So you can have `published` && `inactive` companies, and they should still show up on the frontend, just not on all the charts, because an inactive company is a company that is not getting new assessments, but has historical ones. But we could add an extra scope for when we want both published and active if that makes it more clear.